api/tests/unit/domain/models/Challenge_test.js

```diff
@@ -212,6 +212,48 @@ describe('Unit | Domain | Challenge', () => {
     });
   });
 
+  describe('#get isLive', () => {
+    it('should return true when challenge is valide', () => {
```

Comment:
`is valid` 👀

Reply:
ou alors is "valide" si on garde le nom exact

Reply:
ouais chais pas faut vraiment que je me mette au clair sur cette histoire de franglish